### ISSUE TITLE
Getting "ETHON: performed EASY..." log messages in STDOUT with no information how to silence it.

### ISSUE THREAD

**Issue by swistak** [typhoeus/typhoeus#670]
Every time I do Typhoeus::Request...run I'm getting my console spammed with:
`ETHON: performed EASY effective_url=...response_code=200 return_code=ok total_time=0.283943`

There's no indication how to stop this in documentation.

Setting `Typhoeus::Config.verbose = false` does not work.

How do I stop it from happening? What is ETHON?

**Comment by Jaleelahmed94** [comment]
@swistak, were you able to figure this out?

**Comment by swistak** [comment]
Yes. For anyone who stumbles upon it. `Ethon` seems to be a `curl` wrapper, although I have no ida why it's a separate gem.

Way to silence it is doing something along the lines of: 
`Ethon.logger = Logger.new('log/http_requests.log')` 
in initializer.
